src/index.html

```diff
@@ -5,8 +5,11 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <script type="text/javascript" src="app.js"></script>
-    <script type="text/javascript" src="../node_modules/paper/dist/paper-full.js"></script>
+    <!--<script type="text/javascript" src="../node_modules/paper/dist/paper-full.js"></script>-->
+    <script type="text/javascript" src="libs/paper-full.js"></script>
```

Comment:
We should _**not**_ be including paper into the repo as code, that's the whole point of NPM.